todo_app/trello_api_client.py

```diff
@@ -0,0 +1,90 @@
+import os
+from dotenv import load_dotenv
+import requests
+from todo_app.item import Item
+from todo_app.trello_board import TrelloBoard
+from todo_app.trello_list import TrelloList
+from datetime import datetime
+
+class TrelloAPIClient():
+    def __init__(self):
+        load_dotenv()
```

Comment:
As Jack said you shouldn't need to do this, it's better to load the right .env file in your tests.

You're currently depending on the order of your tests, since if your integration tests run before your e2e tests they'll load .env.test and this will not override it.
e.g. this fails:
```bash
mv todo_app/tests/test_app_integrations.py todo_app/tests/test_app_aarunmefirst_integrations.py
poetry run pytest
```